app/models/shipit/task.rb

```diff
@@ -201,16 +199,16 @@ def run_now!
 
     def write(text)
       log_output(text)
-      chunks.create!(text: text)
+      Shipit.redis.append(output_key, text)
```

Comment:
I'm tempted to set an expiry on that key, just in case something goes wrong. But I'm the same time I'm worried some users might not have output rollup setup properly.

Not too sure what to do.

Reply:
So to answer myself:

  - Currently tasks are eligible for rollup once they are completed and older than 1 hour.
  - The rollup is supposed to be triggered by and hourly cron.

Somehow I don't trust Redis much to keep this data at rest, so I'm thinking we should enqueue a rollup job once the task complete, to make sure we don't keep too much data only in Redis.

Reply:
My initial proposal did include triggering a rollup right after the task finished, but once we decided on the this specific approach, it seemed to me that the hourly cron job plus the eligibility threshold should be enough. Redis should be resilient enough to keep the output for a couple of hours at most.

But it makes sense to be extra sure, and roll the task up after it finishes 👍